bigbluebutton-html5/private/config/settings.yml

```diff
@@ -24,7 +24,7 @@ public:
     # audio. Default value is true
     # Firefox users: if no output devices is shown, you may set the flag
     # "media.setsinkid.enabled" to make it work properly
-    # enableDynamicAudioDeviceSelection: true
+    enableDynamicAudioDeviceSelection: true
```

Comment:
Was this change on purpose?

Reply:
Yes,  in last implementation the code was considering it anyway true if undefined ([code](https://github.com/bigbluebutton/bigbluebutton/blob/v2.6.x-release/bigbluebutton-html5/imports/ui/components/audio/audio-controls/component.jsx#L109)), so I prefered uncomment this part and make it used in the refactor